examples/integrations/oci/vault-reactive/src/main/java/io/helidon/examples/integrations/oci/vault/reactive/OciHandler.java

```diff
@@ -0,0 +1,46 @@
+/*
+ * Copyright (c) 2022 Oracle and/or its affiliates.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package io.helidon.examples.integrations.oci.vault.reactive;
+
+import java.util.function.Consumer;
+
+import com.oracle.bmc.responses.AsyncHandler;
+
+final class OciHandler {
+    private static final System.Logger LOGGER = System.getLogger(OciHandler.class.getName());
```

Comment:
Is this our preferred logging strategy now? I don't see anything one way or the other in the development guidelines so I wasn't sure.

Reply:
Using jul logger. We have an issue to switch to System.Logger, we will do it as a single task.